src/legacy/core_plugins/vis_type_markdown/index.ts

```diff
@@ -18,19 +18,27 @@
  */
 
 import { resolve } from 'path';
-import { Legacy } from '../../../../kibana';
-import { LegacyPluginApi } from '../../plugin_discovery/types';
+import { Legacy } from 'kibana';
 
-// eslint-disable-next-line import/no-default-export
-export default function MarkdownVisTypePlugin(kibana: LegacyPluginApi) {
-  const config: Legacy.PluginSpecOptions = {
+import { LegacyPluginApi, LegacyPluginInitializer } from '../../../../src/legacy/types';
+
+const markdownPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyPluginApi) =>
+  new Plugin({
     id: 'vis_type_markdown',
```

Comment:
Per my note in [this table vis PR](https://github.com/elastic/kibana/pull/41872#discussion_r306999054), would you mind updating the `id` here and in `package.json` to revert to the previous `markdown_vis`? That way we can make these breaking changes at once.